apisix/schema_def.lua

```diff
@@ -402,16 +402,10 @@ local upstream_schema = {
                 },
             },
             dependencies = {
-                client_cert = {
-                    required = {"client_key"},
-                    ["not"] = {required = {"client_cert_id"}}
```

Comment:
Why do you remove the not condition?

Reply:
@soulbird please help to check this

Reply:
The not condition is not required at both places `client_cert` and `client_cert_id`, having the not in clien_cert_id is enough to make both fields mutually exclusive. @monkeyDluffy6017

Reply:
As @Revolyssup said, there should only be the following three cases of the three properties(client_cert_id, client_cert, client_key) here:
1. None of them exist
2. Only client_cert, client_key
3. Only client_cert_id
The modified conditions can ensure that "client_cert" and "client_key" must exist at the same time, and three properties will not exist at the same time.

Reply:
Could you add some test cases to cover these conditions?